dom_wasmjs_gopherjs.go

```diff
@@ -26,6 +26,10 @@ var (
 	undefined = wrappedObject{js.Undefined()}
 )
 
+func init() {
+	js.Global().Call("eval", "$vectyDelete = function(obj, key) { delete(obj[key])}")
```

Comment:
Sorry for the delayed review. I did review this a while back but waited to see what the resolution was for https://github.com/golang/go/issues/33079 and failed to post that over here.

Go 1.14 has added an explicit `Delete` operator that does not require using `eval` like in this PR: (see https://github.com/golang/go/issues/33079) and using `eval` has negative consequences, like needing a certain CSP (see https://github.com/golang/go/issues/26748)

I think the best option here is to use the new `Delete` method and require Go tip until Go 1.14 is released: https://tip.golang.org/pkg/syscall/js/#Value.Delete

If anyone can update this PR to do that, I can merge this.

Reply:
Sent https://github.com/gopherjs/vecty/pull/256 to fix this.